### DISCUSSION TITLE
Gb whatsapp wants update upgrade

### DISCUSSION THREAD

**Discussion by sabir789** [anirudhmalik/xhunter#104] [General]
Hi, whenever i send my payload to victim and when victim open my whatsapp payload the gb whatsapp want upgarde after updating my connection not estiblish to victim plz add update gb whatsapp to the code.

**Comment by anirudhmalik** [comment]
Closing in favor of v2. See [v2.0-demo](https://github.com/anirudhmalik/xhunter/releases/tag/v2.0-demo) and the repo README. New bugs: open a fresh issue with repro steps. Thanks.
